File: src/Squidex.Core/Schemas/SchemaProperties.cs
// ==========================================================================
//  SchemaProperties.cs
//  Squidex Headless CMS
// ==========================================================================
//  Copyright (c) Squidex Group
//  All rights reserved.
// ==========================================================================
namespace Squidex.Core.Schemas
{
    public sealed class SchemaProperties : NamedElementProperties
    {
        public SchemaProperties(
            string label, 
            string hints)
            : base(label, hints)
        {
        }
    }
}

File: src/Squidex/app/features/schemas/pages/schema/schema-properties.ts
/*
 * Squidex Headless CMS
 *
 * @license
 * Copyright (c) Sebastian Stehle. All rights reserved
 */

export class SchemaPropertiesDto {
    constructor(
        public readonly name: string,
        public readonly label: string,
        public readonly hints: string
    ) {
    }
}